Classes/Network/FLEXNetworkSettingsTableViewController.m

```diff
@@ -74,8 +74,12 @@ - (void)cacheLimitAdjusted:(UISlider *)sender
 
 - (void)clearRequestsTapped:(UIButton *)sender
 {
-    UIActionSheet *actionSheet = [[UIActionSheet alloc] initWithTitle:nil delegate:self cancelButtonTitle:@"Cancel" destructiveButtonTitle:@"Clear Recorded Requests" otherButtonTitles:nil];
-    [actionSheet showInView:self.view];
+    UIAlertController *actionSheet = [UIAlertController alertControllerWithTitle:nil message:nil preferredStyle:UIAlertControllerStyleActionSheet];
+    [actionSheet addAction:[UIAlertAction actionWithTitle:@"Cancel" style:UIAlertActionStyleCancel handler:nil]];
+    [actionSheet addAction:[UIAlertAction actionWithTitle:@"Clear Recorded Requests" style:UIAlertActionStyleDestructive handler:^(UIAlertAction * _Nonnull action) {
+        [[FLEXNetworkRecorder defaultRecorder] clearRecordedActivity];
+    }]];
+    [self presentViewController:actionSheet animated:YES completion:nil];
```

Comment:
For action sheet, let's make sure we set the `popoverPresentationController.sourceView` and `popoverPresentationController.sourceRect`.
Otherwise it's going to crash on iPads.

Reply:
Good catch! Updated